saleor/graphql/order/types.py

```diff
@@ -46,9 +47,9 @@ class OrderEvent(CountableDjangoObjectType):
     oversold_items = graphene.List(
         graphene.String, description="List of oversold lines names."
     )
-    lines = graphene.List(OrderEventLineObject, description="The concerned lines.")
+    lines = graphene.List(OrderEventOrderLineObject, description="The concerned lines.")
     fulfilled_items = graphene.List(
-        OrderEventLineObject, description="The lines fulfilled."
+        lambda: FulfillmentLine, description="The lines fulfilled."
```

Comment:
Shouldn't this field be wrapped in `gql_optimizer`? I don't know if it would help us since in the resolver we do queryset filtering.

Reply:
I have a hard time to see how it will optimize it without knowing what to look for?

Reply:
I mean this type is able to return something that requires a database query underneath (to fetch the fulfillment lines). I'm wondering if it wouldn't introduce N+1 problem in `line` or `fulfilledItems` fields when I run a query like: 
```
{
  order(id: "") {
    events {
      lines
      fulfilledItems
    }
  }
}
```

Reply:
Oh okay, I see what you mean! 👍